src/mscorlib/src/System/Random.cs

```diff
@@ -46,11 +47,17 @@ public class Random {
       //
       // Constructors
       //
-    
+      
+      /*=========================================================================================
+      **Action: Initializes a new instance of the Random class, using a default seed value
+      ===========================================================================================*/
       public Random() 
-        : this(Environment.TickCount) {
+        : this(GenerateSeed()) {
       }
-    
+      
+      /*=========================================================================================
+      **Action: Initializes a new instance of the Random class, using a the specified seed value
```

Comment:
Do need these added header comments? If we keep them, the "a the" here should be fixed.